code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -34,7 +34,7 @@
 		qdel(src)
 	return
 
-/obj/structure/stool/MouseDrop(atom/over_object, skip_fucking_stool_shit = 0)
+/obj/structure/stool/MouseDrop(atom/over_object, src_location, over_location, src_control, over_control, params, var/skip_fucking_stool_shit = 0)

```

Comment:
it was mostly to help signify that it was an additional argument, not one that is normally in the proc like the others.